src/main/java/com/hermesworld/ais/galapagos/changes/config/GalapagosChangesConfig.java

```diff
@@ -1,5 +1,6 @@
 package com.hermesworld.ais.galapagos.changes.config;
 
+import com.hermesworld.ais.galapagos.uisupport.controller.ProfilePicture;
```

Comment:
Dependency violation: A "config" package (or, the "changes" "module") should not be dependent on "uisupport" package. Please move ProfilePicture here, in this Config package.